### ISSUE TITLE
Demo not working

### ISSUE THREAD

**Issue by maiieul** [johnymontana/grand-stack-starter-auth0-demo#1]
The demo at [auth-demo.grandstack.io](url) is not working. Is it normal?

**Comment by thelovesmith** [comment]
Yeah like @maieulChevalier  said the demo is not working. After logging in You still can't access the data. Can we please get an update for this demo. This is the only example of authentication in Grand stack that I have been able to find. I really need some help getting authentication and user roles set up with grand stack.. 

this is what is logging in the console 
![Screen Shot 2019-12-03 at 4 25 21 PM](https://user-images.githubusercontent.com/29714476/70091175-93d51500-15e9-11ea-82f7-a036c94bf424.png)
